lms/djangoapps/student_account/views.py

```diff
@@ -343,32 +342,28 @@ def get_user_orders(user):
     )
 
     for order in commerce_user_orders:
-        try:
-            if order['status'].lower() == 'complete':
-                order_lines = order['lines']
-                date_placed = datetime.strptime(order['date_placed'], "%Y-%m-%dT%H:%M:%SZ")
-                order_data = {
-                    'number': order['number'],
-                    'price': order['total_excl_tax'],
-                    'order_date': strftime_localized(
-                        date_placed.replace(tzinfo=pytz.UTC), 'SHORT_DATE'
-                    ),
-                    'receipt_url': EcommerceService().get_receipt_page_url(order['number']),
-                    'lines': order_lines
-                }
-
-                if len(order_lines) == 1:
-                    product = order_lines[0].get('product')
-
-                    if product:
-                        for attribute in product['attribute_values']:
-                            if attribute['name'] == 'certificate_type' and attribute['value'] in allowed_course_modes:
-                                user_orders.append(order_data)
-                else:
-                    user_orders.append(order_data)
-        except KeyError:
-            log.exception('Invalid order structure: %r', order)
-            return no_data
+        if order['status'].lower() == 'complete':
+            date_placed = datetime.strptime(order['date_placed'], "%Y-%m-%dT%H:%M:%SZ")
+            order_data = {
+                'number': order['number'],
+                'price': order['total_excl_tax'],
+                'order_date': strftime_localized(
+                    date_placed.replace(tzinfo=pytz.UTC), 'SHORT_DATE'
+                ),
+                'receipt_url': EcommerceService().get_receipt_page_url(order['number']),
+                'lines': [],
+            }
+
+            # If the order lines contain a product that is not a Seat
```

Comment:
This will throw off order totals. Remind me again what we are trying to hide here and why. If we are trying to hide coupons, the odds of an individual learner buying coupons is really low. Even if they did buy coupons somehow, it's no big deal if we display them. I am inclined to say remove this filter.